app/services/permit_application_export_service.rb

```diff
@@ -5,13 +5,27 @@ def initialize()
   def application_metrics_csv
     CSV.generate(headers: true) do |csv|
       csv << I18n.t("export.application_metrics_csv_headers").split(",")
-      jurisdiciton_counts = PermitApplication.count_by_jurisdiction_and_status
+      jurisdiciton_counts = PermitApplication.stats_by_template_jurisdiction_and_status
       jurisdiciton_counts.each do |jc|
-        jurisdiction_name = jc[:name]
-        draft = jc[:draft]
-        submitted = jc[:submitted]
+        jurisdiction_name = jc[:jurisdiction_name]
+        permit_type = jc[:permit_type]
+        activity = jc[:activity]
+        first_nations = jc[:first_nations]
+        draft_applications = jc[:draft_applications]
+        submitted_applications = jc[:submitted_applications]
+        average_days_spent_before_first_submit = (jc[:average_time_spent_before_first_submit] / 86400.0).round(2)
+        average_days_spent_before_latest_submit = (jc[:average_time_spent_before_latest_submit] / 86400.0).round(2)
```

Comment:
I am assuming the / 86400 is to convert ms or something to days. Can you just a comment here explicitly for future developers for clarity